practices/managing-node-projects.md

```diff
@@ -139,13 +148,13 @@ We _don't_ do this:
 
 ### Classifying "built" run-time dependencies
 
-While the classification of run-time and development dependencies is usually fairly clear-cut, there can be grey-areas.  For example dependencies that are required for the running of your website/app but are processed by a build tool (perhaps to concatenate or transpile them).
+While the classification of run-time and development dependencies is usually fairly clear-cut, there can be grey areas.  For example, dependencies that are required for the running of your website/app but are processed by a build tool (perhaps to concatenate or transpile them).
 
-In these instances it is technically true that the dependency wouldn't need to be installed in production since only the built asset would be served.  However classifying it as a development dependency would mischaracterise its contribution to the application, and could lead it to be treated with less care than it deserves (including with a looser semver range [as above](#examples-1)).
+In these instances it is technically true that the dependency wouldn't need to be installed in production since only the built asset would be served.  However classifying it as a development dependency would mischaracterise its contribution to the application, and could lead it to be treated with less care than it deserves (including with a looser semver range, as per the [previous example](#examples-1)).
 
-In an ideal world where HTTP2, modern JavaScript syntax, and ES6 modules are widely supported, there would be no need to transpile or concatenate any dependencies even for browsers.  Instead, they would simply be loaded using `import` statements and as such those dependencies would be directly served from their installed location (and as such would umabiguiously be run-time dependencies, not development dependencies). When working within a Node.js environment this is already possible through its native support for CommonJS `require` statements.
+In an ideal world where HTTP2, modern JavaScript syntax, and ES6 modules are widely supported, there would be no need to transpile or concatenate JavaScript dependencies for browsers.  Instead, they would simply be loaded using `import` statements and would be directly served from their installed location. Therefore they would unambiguously be run-time dependencies, not development dependencies. (When working within a Node.js environment this is already possible through its native support for CommonJS `require` statements.)
```

Comment:
>Instead, they would simply be loaded using `import` statements and would be directly served from their installed location.

concision, contractions: "Instead, they'd simply be loaded using `import` statements and directly served from their installed location." 

>Therefore they would unambiguously be...

contractions: "Therefore they'd unambiguously be..."

>When working within a Node.js environment...

concision, active voice: "In a Node.js environment..." The parentheses probably aren't needed either tbh